src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -62,7 +63,10 @@ public class RoatpApplicationController : Controller
             ISessionService sessionService, IConfigurationService configService, IUserService userService, IUsersApiClient usersApiClient,
```

Comment:
This controller has lots of dependencies. It's getting out of control.... 😢

Reply:
As discussed, will raise a tech debt story to review this, splitting up and removing redundant code inherited from EPAO